match/grand_isle_pd.py

```diff
@@ -1,3 +1,44 @@
+<<<<<<< HEAD
```

Comment:
merge conflict.

Reply:
No conflict errors are appearing for me.

Reply:
Found them. Conflicts are merged.

Reply:
Please resolve conflicts here too.

Reply:
Removed from this pull request. Duplicate of the clean and match/grand isle updates in the "bug/events" pull request.